app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CommentsController < ApplicationController
+  def index
+    @post = Post.find(params[:post_id])
+    @comments = @post.comments
+    render json: @comments
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new(comment_params)
+    @comment.author = current_user
+
+    if @comment.save
+      render json: @comment
+    else
+      render json: { errors: @comment.errors.full_messages }, status: :unprocessible_entity
+    end
+  end
```

Comment:
- [ Required ] I think it would be nice if you ensure that it's possible to add a `comment` to a `post` via the `http://localhost:3000/api/v1/users/1/posts/1/comments` endpoint. Currently, when I try to add a post via this endpoint I get the error below.

## Error when adding a post
![Screenshot from 2023-10-06 12-38-49](https://github.com/MNisarAli/blog_app/assets/16392046/747274e4-9930-422a-bf4c-32549e773e68)